src/experimentExtractor/index.js

```diff
@@ -0,0 +1,128 @@
+const fs = require("fs-extra");
+const {
+  EXPERIMENT_COMPONENT_NAME,
+  EXPERIMENTS_FILE_NAME,
+} = require("./constants");
+const { extractExperimentData } = require("./parser");
+
+// TODO: move to utils and add tests
+const chunkNameToPageNameRegex = (chunkName) => {  
+  // replacing all \\ to a /
+  let normalizedChunkName = chunkName.replace(/\\+/gi, "/");
+  // remove .js extension 
+  normalizedChunkName = normalizedChunkName.replace(/\.\S+$/gi, "");
+  // replace 'pages' prefix with ^
+  normalizedChunkName = normalizedChunkName.replace(/^pages/gi, "^");
+  // replace all / with \/
+  normalizedChunkName = normalizedChunkName.replace(/\//gi, "\\/");
+  // replace [__anything__] with [^\s\/]+
+  normalizedChunkName = normalizedChunkName.replace(/\[\S+\]/gi, "[^\\s\\/]+");
+  // add $ at the end
+  normalizedChunkName = `${normalizedChunkName}$`
+  return normalizedChunkName;
+};
```

Comment:
Please either use more descriptive names, or otherwise chain `.replace` calls:

```js
return chunkName
  // replace all `\\` with `/`
  .replace(/\\+/gi, '/')
  // remove file extension
  .replace(/\.\S+$/gi, '/')
  // ...
```

Reply:
Done